apps/codeforafrica/src/components/ArticleCardList/ArticleCardList.js

```diff
@@ -23,31 +23,31 @@ const ArticleCardList = React.forwardRef(function ArticleCardList(props, ref) {
       ref={ref}
     >
       {articles?.map((article) => (
-        <Grid item xs={12} sm={4} key={article.href}>
+        <Grid item xs={12} sm={4} key={article.id}>
           <ArticleCard
             sx={{
               width: "100%",
               height: "100%",
             }}
           >
             <CardActionArea
-              component={article.href ? Link : undefined}
-              href={article.href}
+              component={article.slug ? Link : undefined}
+              href={`/stories/${article.slug}`}
```

Comment:
Can we add this in the lib? `ArticleXXX` components are designed to be reusable for both opportunities and stories.

Reply:
@kilemensi resolving this, and I think it could be much better to do it in the `lib/index.js` because we need to either append either stories or opportunites to the slug, allowing us to differentiate between the two.

Reply:
Sure @kelvinkipruto , in `lib` or in `api.ghost` since we we know primary tags.